src/utils/embedded_file_manager.cr

```diff
@@ -1,5 +1,5 @@
-module EmbeddedFileManager 
+module EmbeddedFileManager
   macro cluster_tools
-    CLUSTER_TOOLS = Base64.decode_string("{{ `cat ./tools/cluster-tools/manifest.yml | base64` }}")
+    CLUSTER_TOOLS = ENV["USE_MANIFEST_FOR_SPECS"]? ? File.read("./tools/cluster-tools/manifest.yml") : File.read("./lib/cluster_tools/tools/cluster-tools/manifest.yml")
```

Comment:
`USE_MANIFEST_FOR_SPECS` is a confusing name with this usage.

how about `USE_MANIFEST_SPECS_DIRECTORY`

also I wonder if we could compile this file into the binary for the shard...